config/admin/bootstrap.go

```diff
@@ -36,12 +42,25 @@ var fs embed.FS
 // Bootstrap creates CRDs and the resources in this package by continuously retrying the list.
 // This is blocking, i.e. it only returns (with error) when the context is closed or with nil when
 // the bootstrapping is successfully completed.
-func Bootstrap(ctx context.Context, crdClient apiextensionsclient.Interface, dynamicClient dynamic.Interface) error {
-	return confighelpers.Bootstrap(ctx, crdClient, dynamicClient, fs, []metav1.GroupResource{
-		{Group: tenancy.GroupName, Resource: "workspaces"},
+func Bootstrap(ctx context.Context, crdClient apiextensionsclient.Interface, discoveryClient discovery.DiscoveryInterface, dynamicClient dynamic.Interface) error {
+	crds := []metav1.GroupResource{
```

Comment:
maybe a comment why we want them here would make sense.

Reply:
Added in upcoming commit